1040.rb

```diff
@@ -0,0 +1,26 @@
+n1 = gets.to_f
+n2 = gets.to_f
+n3 = gets.to_f
+n4 = gets.to_f
+
+media = (2*n1 + 3*n2 + 4*n3 + 1*n4)/10
+mediar = media.round(1)
```

Comment:
```suggestion
media_arredondada = media.round(1)
```

Seria interessante evitar abreviar muito os nomes, dessa forma você tem um código mais legível e que representa de fato o que você está fazendo.